### PR TITLE
Make the kitchensink button removable from plugins

### PR DESCRIPTION
**Pull request by azaozz**
See #10963.

Tested with the code there, also with plugins. Ideally the `kitchensink` button would be named the same as in the classic editor: `wp-adv`, but that brings some problems with `/js/tinymce/plugins/wordpress`.

### CODE REVIEW

**Comment by azaozz** [comment]
@iseulde hmm, or perhaps we should use the old name, `wp-adv`,  to remove any possibility of regressions and tweak the wp-includes/js/tinymce/plugins/wordpress plugin a bit to make it work properly in the classic block. I'll open a corresponding ticket for core.

**Comment by azaozz** [comment]
This is a "blocker" for WP 5.0. This PR is the very minimum that can be done to fix it. 

Or perhaps we should handle this in core trac? There we can tweak the existing `wordpress` plugin and change how it handles the `wp-adv` button to account for the use in the classic block? We can even implement the same way of hiding/showing the extra toolbar(s), by toggling a class on the container div.

File: packages/block-library/src/classic/edit.js
Comment (iseulde) [inline packages/block-library/src/classic/edit.js]: What is this doing?

**Comment by iseulde** [comment]
Let's do this, it looks good. I'll test in a bit and approve. :)

**Review by ellatrix — APPROVED** [review]

**Comment by azaozz** [comment]
Note: need to update the `wordpress` TinyMCE plugin in core a bit for this to work.

**Comment by youknowriad** [comment]
Anything blocking merge here @azaozz @iseulde?

**Comment by azaozz** [comment]
This went through few changes but "came back" to the initial code. Should be OK to merge to Gutenberg. https://core.trac.wordpress.org/attachment/ticket/45264/45264.3.diff is needed to fix the regression for core.

**Comment by youknowriad** [comment]
@iseulde Would you mind taking another look before merge?

**Comment by youknowriad** [comment]
Moving forward here. @iseulde is AFK for some days.

**Pull request merged**
